docs/Architecture/ADR-0001 Use Flutter.md

```diff
@@ -0,0 +1,26 @@
+# Use Flutter
+
+## Status
+
+Accepted
+
+## Context
+
+Flutter is a framework for building apps provided by google.  Google desribe it as a
+> ... UI toolkit for building beautiful, natively compiled applications for mobile, web, and desktop from a single codebase.
+
+It places an emphasis on fast development, expressive and flexible ui, and native performance.
```

Comment:
some general housekeeping. Some mentions of `Flutter` need to be spelt with a capital F so that we’re respectful to all the Flutterers out there 🙂